apps/prairielearn/src/pages/home/home.html.ts

```diff
@@ -55,7 +55,6 @@ export function Home({
         </header>
 
         <main id="content" class="flex-grow-1">
-          <h1 class="sr-only">PrairieLearn Homepage</h1>
```

Comment:
Why was this removed?

Reply:
Removed inadvertently when testing screen reader functionality. Restored in [c5187bb](https://github.com/PrairieLearn/PrairieLearn/pull/10483/commits/c5187bb79e07a5a99f75ef5cfa5a9961d6781e2d).